src/components/SearchAndFilter/SearchAndFilter.js

```diff
@@ -153,13 +184,25 @@ const SearchAndFilter = ({
           aria-hidden={filterPanelHidden}
         >
           <ContextualMenu>
+            {searchTerm.length > 0 && (
+              <div
+                className="search-prompt"
+                onClick={() => handleSubmit()}
+                onKeyPress={(e) => searchPromptKeyPress(e)}
```

Comment:
[onKeyPress is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onkeypress)